[ILM] Add support for frozen phase in UI

jloleysens opened the pull request:
## Summary

Support configuring the frozen phase in both server and UI of ILM. This also updates the index management plugins extension to support querying for "frozen" phase in the index table.

## How to test

1. Create a new ILM policy that consists of Hot and Frozen phases only (to simplify testing). In the UI, make sure that you use all options for frozen phase except for setting required replicas as this will cause the phase to get stuck there if the cluster cannot create a replica of the shards.
2. Follow this [tutorial](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started-index-lifecycle-management.html) to set up an index template that will use the ILM policy and  create an initial write index (making sure to keep your naming consistent). Make sure to set any timing values to one second (`1 second`) so that you do not have to wait for phase rollover too long.
3. Run the following in Dev Tools to make ILM check progress more frequently:
```
PUT _cluster/settings
{
  "transient": {
    "indices.lifecycle.poll_interval": "5s"  
  }
}
```

Make sure to set this back to the default of `10m` so that ILM doesn't continue doing unnecessary work after testing.

What you should see in index management is new indices being created every 5 seconds as rollover happens and they should have the "Frozen" label applied to them in the index management UI.

Fix https://github.com/elastic/elasticsearch/issues/61345

## Screenshots

<img width="1203" alt="Screenshot 2020-08-26 at 12 56 47" src="https://user-images.githubusercontent.com/8155004/91295622-a573d380-e79b-11ea-8b26-03d809d7a2be.png">


### Checklist

Delete any items that are not applicable to this PR.

- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios

elasticmachine commented:
Pinging @elastic/es-ui (Team:Elasticsearch UI)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/)
* Commit: 19afe82eb5f2c0844121676d6190b0349f43b42d
* This comment will update when the build is complete

### Failed CI Steps
* [Check i18n](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/execution/node/370/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/","number":"70464","commit":"19afe82eb5f2c0844121676d6190b0349f43b42d"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/)
* Commit: 9ff7eaaa72bec33bc365d3cea0ba740a26f16591
* This comment will update when the build is complete

### Failed CI Steps
* [Check i18n](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/execution/node/394/log)

### History
* :broken_heart: [Build #70464](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/) failed 19afe82eb5f2c0844121676d6190b0349f43b42d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/","number":"70464","commit":"19afe82eb5f2c0844121676d6190b0349f43b42d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/","number":"70477","commit":"9ff7eaaa72bec33bc365d3cea0ba740a26f16591"}]}
PIPELINE-->

jloleysens commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/)
* Commit: 5149dd2f1e907c763089a82b8102572cb3fe1091

### Build metrics
#### @kbn/optimizer bundle module count
| id | [value](https://github.com/elastic/kibana/commit/5149dd2f1e907c763089a82b8102572cb3fe1091) | diff | [baseline](https://github.com/elastic/kibana/commit/a358c5768ea4f378270140d4e480fece98e8e103) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 159 | +2 | 157 |

#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/5149dd2f1e907c763089a82b8102572cb3fe1091) | diff | [baseline](https://github.com/elastic/kibana/commit/a358c5768ea4f378270140d4e480fece98e8e103) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 275.7KB | +16.0KB | 259.7KB |

#### page load bundle size
| id | [value](https://github.com/elastic/kibana/commit/5149dd2f1e907c763089a82b8102572cb3fe1091) | diff | [baseline](https://github.com/elastic/kibana/commit/a358c5768ea4f378270140d4e480fece98e8e103) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 230.2KB | +430.0B | 229.8KB |

#### distributable file count
| id | [value](https://github.com/elastic/kibana/commit/5149dd2f1e907c763089a82b8102572cb3fe1091) | diff | [baseline](https://github.com/elastic/kibana/commit/a358c5768ea4f378270140d4e480fece98e8e103) |
| --- | --- | --- | --- |
| total | 53237 | +2 | 53235 |

### History
* :broken_heart: [Build #70477](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/) failed 9ff7eaaa72bec33bc365d3cea0ba740a26f16591
* :broken_heart: [Build #70464](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/) failed 19afe82eb5f2c0844121676d6190b0349f43b42d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/","number":"70464","commit":"19afe82eb5f2c0844121676d6190b0349f43b42d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/","number":"70477","commit":"9ff7eaaa72bec33bc365d3cea0ba740a26f16591"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/","number":"70777","commit":"5149dd2f1e907c763089a82b8102572cb3fe1091"}]}
PIPELINE-->

yuliacech commented:
Hi @jloleysens , 
I tested the code locally with a data stream and the frozen phase worked for me. 
I noticed some issues with validation and typings, I will add comments below.

yuliacech commented on x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/phases/frozen_phase.tsx:
```diff
@@ -0,0 +1,210 @@
+          <Fragment>
+            {phaseData.phaseEnabled ? (
+              <Fragment>
+                <MinAgeInput<FrozenPhaseInterface>
```
`MinAgeInput`'s props are currently typed to accept warm, cold and delete phases. I'm not sure why the type check is not picking it up, but it might be good to add frozen phase to props generics.

yuliacech commented on x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/phases/frozen_phase.tsx:
```diff
@@ -0,0 +1,210 @@
+                />
+                <EuiSpacer />
+
+                <NodeAllocation<FrozenPhaseInterface>
```
Similar here, the props generics don't include frozen phase, but  type check is not showing it.

yuliacech commented on x-pack/plugins/index_lifecycle_management/public/application/services/policies/policy_validation.ts:
```diff
@@ -148,6 +158,7 @@ export const validatePolicy = (
   const hotPhaseErrors = validateHotPhase(policy.phases.hot);
   const warmPhaseErrors = validateWarmPhase(policy.phases.warm);
   const coldPhaseErrors = validateColdPhase(policy.phases.cold);
+  const frozenPhaseErrors = validateColdPhase(policy.phases.frozen);
```
I think `validateFrozenPhase` function should be used here, but then again, it's weird why type_check didn't show any errors.

jloleysens replied:
TS uses structural type checking(https://www.typescriptlang.org/docs/handbook/type-compatibility.html). Structurally frozen and cold are the same at the moment which is why TS is not complaining. But you are right this should be aligned with their own functions/types as this is copy/pasta mistake.

yuliacech commented on x-pack/plugins/index_lifecycle_management/public/application/services/policies/policy_validation.ts:
```diff
@@ -162,6 +173,7 @@ export const validatePolicy = (
       hot: hotPhaseErrors,
       warm: warmPhaseErrors,
       cold: coldPhaseErrors,
+      frozen: frozenPhaseErrors,
```
I think `Object.keys(frozenPhaseErrors).length === 0` should be added to `isValid` variable calculation, otherwise the form will be submitted even when there are validation errors in frozen phase.

jloleysens commented:
@elasticmachine merge upstream

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/)
* Commit: 067d25a69cc99635953bbcc9e5ebdb56798eeb8f
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/execution/node/386/log)

### History
* :green_heart: [Build #70777](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/) succeeded 5149dd2f1e907c763089a82b8102572cb3fe1091
* :broken_heart: [Build #70477](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/) failed 9ff7eaaa72bec33bc365d3cea0ba740a26f16591
* :broken_heart: [Build #70464](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/) failed 19afe82eb5f2c0844121676d6190b0349f43b42d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/","number":"70464","commit":"19afe82eb5f2c0844121676d6190b0349f43b42d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/","number":"70477","commit":"9ff7eaaa72bec33bc365d3cea0ba740a26f16591"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/","number":"70777","commit":"5149dd2f1e907c763089a82b8102572cb3fe1091"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/","number":"70817","commit":"067d25a69cc99635953bbcc9e5ebdb56798eeb8f"}]}
PIPELINE-->

jloleysens commented:
@yuliacech Thanks for the review and great catch on frozen phase validation! I've addressed your PR feedback, would you mind taking another look?

I did a slight restructure to the phase types. Because there is so much overlap I thought it would be easiest to use inheritance for the interfaces. So now when we create a new "shared" form UI component when can just extend the relevant phases and structurally the phases that extend that type should fit into the component type requirements. Let me know what you think!

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70824/)
* Commit: c366c5ec8099f29351bf1840fb43ec59fb5ef280

### Build metrics
#### @kbn/optimizer bundle module count
| id | [value](https://github.com/elastic/kibana/commit/c366c5ec8099f29351bf1840fb43ec59fb5ef280) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 159 | +2 | 157 |

#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/c366c5ec8099f29351bf1840fb43ec59fb5ef280) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 275.8KB | +16.1KB | 259.7KB |

#### page load bundle size
| id | [value](https://github.com/elastic/kibana/commit/c366c5ec8099f29351bf1840fb43ec59fb5ef280) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 230.2KB | +430.0B | 229.8KB |

#### distributable file count
| id | [value](https://github.com/elastic/kibana/commit/c366c5ec8099f29351bf1840fb43ec59fb5ef280) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| total | 53126 | +2 | 53124 |

### History
* :broken_heart: [Build #70817](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/) failed 067d25a69cc99635953bbcc9e5ebdb56798eeb8f
* :green_heart: [Build #70777](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/) succeeded 5149dd2f1e907c763089a82b8102572cb3fe1091
* :broken_heart: [Build #70477](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/) failed 9ff7eaaa72bec33bc365d3cea0ba740a26f16591
* :broken_heart: [Build #70464](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/) failed 19afe82eb5f2c0844121676d6190b0349f43b42d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/","number":"70464","commit":"19afe82eb5f2c0844121676d6190b0349f43b42d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/","number":"70477","commit":"9ff7eaaa72bec33bc365d3cea0ba740a26f16591"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/","number":"70777","commit":"5149dd2f1e907c763089a82b8102572cb3fe1091"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/","number":"70817","commit":"067d25a69cc99635953bbcc9e5ebdb56798eeb8f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70824/","number":"70824","commit":"c366c5ec8099f29351bf1840fb43ec59fb5ef280"}]}
PIPELINE-->

yuliacech reviewed (approved):
Hi @jloleysens , 
Great addition to policy types with common phase interfaces! Tested locally and all looks good to me.
I just found a copy-paste error from my commit, so I wont block you on it. But if you want to fix it with this PR, here is the info: both `validateColdPhase` and `validateFrozenPhase` when checking `phase.selectedMinimumAge` actually setting `phaseErrors.phaseIndexPriority`.

jloleysens commented:
Thanks Yulia! I will include the fix and merge once CI is green.

cjcenizal commented on x-pack/plugins/index_lifecycle_management/public/application/services/policies/frozen_phase.ts:
```diff
@@ -0,0 +1,161 @@
+    }
+  }
+
+  if (
```
I would love to see these kinds of rules extracted into functions that have descriptive names, a big block comment explaining the intention (with links to relevant docs), and then covered with unit tests to define the desired behavior. I feel like this would make the code easier to scan, and it'd also be easier to dig into them to understand the _why_ behind these rules. Anybody else feel similarly?

sebelga replied:
That would be indeed very helpful 👍

jloleysens replied:
I agree, I think we should also move the files to directory called `lib` in that process that lives in `application`. For now, the logic is a carbon-copy of that for cold phase, so at least we know it has been working. I will open an issue for this!

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70973/)
* Commit: dc2b0d1c56dfb43074abdff65d1e97a5080567e2

### Build metrics
#### @kbn/optimizer bundle module count
| id | [value](https://github.com/elastic/kibana/commit/dc2b0d1c56dfb43074abdff65d1e97a5080567e2) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 159 | +2 | 157 |

#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/dc2b0d1c56dfb43074abdff65d1e97a5080567e2) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 275.8KB | +16.1KB | 259.7KB |

#### page load bundle size
| id | [value](https://github.com/elastic/kibana/commit/dc2b0d1c56dfb43074abdff65d1e97a5080567e2) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 230.2KB | +430.0B | 229.8KB |

#### distributable file count
| id | [value](https://github.com/elastic/kibana/commit/dc2b0d1c56dfb43074abdff65d1e97a5080567e2) | diff | [baseline](https://github.com/elastic/kibana/commit/8e2cb5684e8d0a75da8762480a17e5800bfe77fb) |
| --- | --- | --- | --- |
| total | 53126 | +2 | 53124 |

### History
* :green_heart: [Build #70824](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70824/) succeeded c366c5ec8099f29351bf1840fb43ec59fb5ef280
* :broken_heart: [Build #70817](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/) failed 067d25a69cc99635953bbcc9e5ebdb56798eeb8f
* :green_heart: [Build #70777](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/) succeeded 5149dd2f1e907c763089a82b8102572cb3fe1091
* :broken_heart: [Build #70477](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/) failed 9ff7eaaa72bec33bc365d3cea0ba740a26f16591
* :broken_heart: [Build #70464](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/) failed 19afe82eb5f2c0844121676d6190b0349f43b42d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70464/","number":"70464","commit":"19afe82eb5f2c0844121676d6190b0349f43b42d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/","number":"70477","commit":"9ff7eaaa72bec33bc365d3cea0ba740a26f16591"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/","number":"70777","commit":"5149dd2f1e907c763089a82b8102572cb3fe1091"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/","number":"70817","commit":"067d25a69cc99635953bbcc9e5ebdb56798eeb8f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70824/","number":"70824","commit":"c366c5ec8099f29351bf1840fb43ec59fb5ef280"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70973/","number":"70973","commit":"dc2b0d1c56dfb43074abdff65d1e97a5080567e2"}]}
PIPELINE-->

jloleysens commented:
@elasticmachine merge upstream

jloleysens commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/71180/)
* Commit: a087b041919328027deed15858b5009f2684db8f

### Build metrics
#### @kbn/optimizer bundle module count
| id | [value](https://github.com/elastic/kibana/commit/a087b041919328027deed15858b5009f2684db8f) | diff | [baseline](https://github.com/elastic/kibana/commit/5f781dcfecc9bcd96d9fa5ecfe5cd3e1cdedf094) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 159 | +2 | 157 |

#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/a087b041919328027deed15858b5009f2684db8f) | diff | [baseline](https://github.com/elastic/kibana/commit/5f781dcfecc9bcd96d9fa5ecfe5cd3e1cdedf094) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 276.0KB | +16.1KB | 259.9KB |

#### page load bundle size
| id | [value](https://github.com/elastic/kibana/commit/a087b041919328027deed15858b5009f2684db8f) | diff | [baseline](https://github.com/elastic/kibana/commit/5f781dcfecc9bcd96d9fa5ecfe5cd3e1cdedf094) |
| --- | --- | --- | --- |
| indexLifecycleManagement | 230.2KB | +430.0B | 229.8KB |

### History
* :green_heart: [Build #70973](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70973/) succeeded dc2b0d1c56dfb43074abdff65d1e97a5080567e2
* :green_heart: [Build #70824](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70824/) succeeded c366c5ec8099f29351bf1840fb43ec59fb5ef280
* :broken_heart: [Build #70817](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/) failed 067d25a69cc99635953bbcc9e5ebdb56798eeb8f
* :green_heart: [Build #70777](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/) succeeded 5149dd2f1e907c763089a82b8102572cb3fe1091
* :broken_heart: [Build #70477](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/) failed 9ff7eaaa72bec33bc365d3cea0ba740a26f16591

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70477/","number":"70477","commit":"9ff7eaaa72bec33bc365d3cea0ba740a26f16591"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70777/","number":"70777","commit":"5149dd2f1e907c763089a82b8102572cb3fe1091"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70817/","number":"70817","commit":"067d25a69cc99635953bbcc9e5ebdb56798eeb8f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70824/","number":"70824","commit":"c366c5ec8099f29351bf1840fb43ec59fb5ef280"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70973/","number":"70973","commit":"dc2b0d1c56dfb43074abdff65d1e97a5080567e2"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/71180/","number":"71180","commit":"a087b041919328027deed15858b5009f2684db8f"}]}
PIPELINE-->
